templates/listing-page-filters.twig

```diff
@@ -0,0 +1,17 @@
+<div class="listing-page-filters">
+    <label for="categories" class="visually-hidden">
+        {{ __('Filter by category', 'planet4-master-theme' ) }}
+    </label>
+    <select class="form-select listing-page-select" id="category">
```

Comment:
Should we use a more specific ID just in case something else is named as "category" in the future?

Reply:
We can, I just don't see what else we could add there since this page is literally just a list of posts, but if you are worried about it I don't mind renaming it `category-filter` for example

Reply:
Don't worry, let's keep "category"